packages/tools/pluggable-widgets-tools/package.json

```diff
@@ -111,7 +111,6 @@
     "zip-a-folder": "^0.0.12"
   },
   "devDependencies": {
-    "@types/xml2js": "^0.4.5",
```

Comment:
I am pretty sure we are using this typings in the typing generator and also on Jordan's  crafting project. Please be careful removing dependencies